elastic/beats
# Include Npcap in Packetbeat when bundled with Agentbeat

## Proposed commit message

Extracting image selector logic into separate package so it can be reused when compiling agent(beat). 
Otherwise npcap installer is not present and packetbeat fails to work when `interfaces!=nil`(trying to install and missing installer)

## Checklist



## How to test this PR locally

Package Agentbeat using `NPCAP_LOCAL=true SNAPSHOT=true PLATFORMS=windows/amd64 PACKAGES=zip mage package` 
run `./agentbeat packetbeat run -c packetbeat.yml` on windows (default packetbeat config downloaded from [here](https://github.com/elastic/beats/blob/main/x-pack/packetbeat/packetbeat.yml) is fine) it should complain `any` is not supported on windows. Error `failed to get device list: couldn't load wpcap.dll` should not be present 

This needs to be packaged on Linux system with [msitools](https://launchpad.net/ubuntu/+source/msitools) installed

## Related issues
Closes https://github.com/elastic/elastic-agent/issues/6108

cc @blakerouse
